pkgs/development/tools/yarn-berry/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub, lib, nodejs, stdenv, yarn }:
+
+stdenv.mkDerivation rec {
+  name = "yarn-berry";
+  version = "3.4.1";
+
+  src = fetchFromGitHub {
+    owner = "yarnpkg";
+    repo = "berry";
+    rev = "@yarnpkg/cli/${version}";
+    hash = "sha256-eBBB/F+mnGi93Qf23xgt306/ogoV76RXOM90O14u5Tw=";
+  };
+
+  buildInputs = [
+    nodejs
+    yarn
```

Comment:
yarn should go to nativeBuildInputs

Reply:
Fixed in 05f304499fdc973573c3cff68ee20ac659e2d46c
Could you take another look please?